workorders.lic

```diff
@@ -246,6 +247,9 @@ class WorkOrders
     stow_hands
 
     quantity.times do |count|
+      if (info['stock-volume'] - (count * recipe['volume'])) > 0
```

Comment:
💡 I'd like to see `(info['stock-volume'] - (count * recipe['volume'])` extracted into a well named var to give more context to the next dev that comes along